external-api/src/constants/regex.constants.ts

```diff
@@ -1,6 +1,6 @@
-export const regexes = {
+export const REGEX = {
   POSTCODE: /^[A-Za-z]{1,2}[0-9Rr][0-9A-Za-z]?[0-9][ABD-HJLNP-UW-Zabd-hjlnp-uw-z]{2}$/,
-  COMPANIES_HOUSE_NUMBER: /^\d{8,10}$/,
+  COMPANIES_HOUSE_NUMBER_REGEX: /^[0-9A-Z]{6,8}$/,
```

Comment:
Why is the company regex not same as `/^(([A-Z]{2}|[A-Z]\d{1}|\d{2})(\d{5,6}|\d{4,5}[A-Z]))$/;`?